packages/configs/default/index.json

```
@@ -10,6 +10,8 @@
       "@parcel/transformer-js",
       "@parcel/transformer-react-refresh-wrap"
     ],
+    "manifest.json": ["@parcel/transformer-webext", "@parcel/transformer-json"],
```

```suggestion
    "manifest.json": ["@parcel/transformer-webext"],
```

The json transformer isn't needed here

---

It actually is useful in case someone has a file named `manifest.json` that isn't a web app manifest. Thoughts?